app/i18n/en.json

```diff
@@ -651,7 +651,8 @@
   "errors": {
     "generic": "There was an error. Please try again later",
     "invalidEmail": "Invalid email",
-    "invalidPhoneNumber": "is not a valid phone number"
+    "invalidPhoneNumber": "is not a valid phone number",
+    "tooManyRequestsPhoneCode": "Please wait 3 hours before requesting another text message"
```

Comment:
I would add "Too many requests. Please wait...", so that they know why they have to wait.

Reply:
This will also be different depending on whether the block is for an ip (1 hour) or phone number (3 hours). We should either just do "Pleas wait before..." and not include the number or pass a different number to the message depending on which error code was hit.

Reply:
I think a user will most likely be using the same phone number and same IP address with each request, in which case both will trigger.  I'm assuming the rate limiting defaults to the longest block duration if 2 are triggered (maybe I'm wrong?), in which case it will **almost** always be 3 hours.  As I said in my comment on the PR itself I think we should change these values anyway (the block durations) as they seem a bit extreme.

Also regarding the error code I couldn't find the error code on the object which we get in the client code.  The object we are dealing with only seems to have a `name` and `message`.  I think it's one of [these](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error).

Reply:
Ah yes, we don't have codes for these mutations. I mean we may need different messages we can check against. You're right that it will probably default to three hours for now unless the user switches ip or we change the values on the backend.

Reply:
I've updated the error message with the "Please wait before..." party you suggested.  I have changed it to say 3 hours instead of 60 minutes as I think it will almost always hit the 3 hour limit.

Reply:
the time mention here (1h or 3h) is a default settings. it can be overriden with a yaml file so we should not have any statics time set in the translation file (we may already have different value for the Bitcoin Beach wallet)

Reply:
I agree with Nicolas. Until we have a standard way of passing back how long the the timeout is in the error, we should just leave the message as "Too many requests. Please wait before requesting another text message."